JiraReporter/Services/ReportEmailerJira.cs

```diff
@@ -84,18 +84,18 @@ private void AddMailRecipients(MailMessage message)
                 message.To.Add(addr);
         }
 
-        private void AddMailAttachments(MailMessage message)
-        {
-            if (Report.IsIndividualDraft)
-                AddAttachementImage(Author.Image, Author.AvatarId, message);
-            else
-            {
-                foreach (var author in Authors)
-                    if(!author.IsEmpty)
-                       AddAttachementImage(author.Image, author.AvatarId, message);
-                AddAttachementImageFromDisk(message);
-            }
-        }
+        //private void AddMailAttachments(MailMessage message)
+        //{
+        //    if (Report.IsIndividualDraft)
+        //        AddAttachementImage(Author.DRTAvatarLink, Author.AvatarId, message);
+        //    else
+        //    {
+        //        foreach (var author in Authors)
+        //            if(!author.IsEmpty)
+        //               AddAttachementImage(author.DRTAvatarLink, author.AvatarId, message);
+        //        AddAttachementImageFromDisk(message);
+        //    }
+        //}
```

Comment:
Metoda asta nu e nevoie sa o comentezi. E suficient ca ai comentat apelul catre ea. 

Daca vrei sa o comentezi inseamna ca ea nu mai e folosita si trebuie stearsa. 

Deci ori o lasi necomentata in ideea in care ar putea fi folosita in viitor chiar daca momentan nu mai este, ori o stergi.